.changes/v3.14.0/1292-improvements.md

```diff
@@ -0,0 +1,2 @@
+* Resources and data sources `vcd_vapp_vm`/`vcd_vm` support IPv6 via `secondary_ip_allocation_mode`
```

Comment:
Nit, just for extra clarity (less questions):

```suggestion
* Resources and data sources `vcd_vapp_vm` and `vcd_vm` support IPv6 via `secondary_ip_allocation_mode`
```

Reply:
Fixed.